docs_dev/releasing.md

```diff
@@ -156,6 +158,9 @@ Use [this script](https://github.com/jlewi/kubeflow-dev/blob/master/create_conte
 create_context.sh $(kubectl config current-context) kubeflow-releasing
```

Comment:
Can you delete the section about building TFServing images please; that step is no longer needed.